src/packages/core/section/section-views/section-views.element.ts

```diff
@@ -13,6 +13,7 @@ import {
 import { createExtensionElement } from '@umbraco-cms/backoffice/extension-api';
 import { UmbLitElement } from '@umbraco-cms/internal/lit-element';
 import { UmbObserverController } from '@umbraco-cms/backoffice/observable-api';
+import { pathFolderName } from 'src/shared/utils/path-folder-name.function.js';
```

Comment:
Action fails on this line. The import path should use an alias.

Reply:
Wups, Corrected :-)